### DISCUSSION TITLE
Error sending confirmation mail

### DISCUSSION THREAD

**Discussion by amauriEbrai** [supabase/supabase#17414] [Questions]
I'm facing a problem since yesterday. 

When I try to signUp a user I get this response :

{
    "code": 500,
    "msg": "Error sending confirmation mail",
    "error_id": "8066b565c21bb374-PRG"
}

It was working fine before so I don't think the problem is on the client side. 
Any thoughts about the error ? 

Here is my function :

```
const handleSubmit = async (e) => {
        e.preventDefault();
        validatePassword(password);
        if (step01 && step02 && step03 && step04 && step05 && step06 && emailSubmitted) {
            let userEmail = email;
            let userPassowrd = password;

            const { data, error } = await supabase.auth.signUp({
                email: userEmail,
                password: userPassowrd,
                options: {
                    emailRedirectTo: cleanedURL,
                }
            });

            if (data) {
                console.log("User created:", data);
                localStorage.setItem('supabase-user', JSON.stringify(data));
                dispatch(advanceStep());
            }

            if (error) {
                console.error("Error creating user:", error.message);
            }

        } else {
            return null;
        }
    }
```

**Comment by GaryAustin1** [comment]
Are you hitting the email rate limiting? I believe still 4 per hour for the default SMTP provider.
Otherwise look in the Auth and Postgres logs for errors.

**Reply by amauriEbrai** [reply]
No, we have a paid plan so that's not the problem.  
I get the error when I enable the SMTP and then when I try to signUp a user I get this:

{
    "code": 500,
    "msg": "Error sending confirmation mail",
    "error_id": "806fbb69b5c6b33f-PRG"
}

**Reply by GaryAustin1** [reply]
Even with a paid plan the limit is 4 for the Supabase SMTP default.  Are there any errors in the auth log of the dashboard?
